crates/uv/tests/it/init.rs

```diff
@@ -446,6 +446,91 @@ fn init_library() -> Result<()> {
     Ok(())
 }
 
+/// Test the uv build backend with using `uv init --lib --preview`. To be merged with the regular
+/// init lib test once the uv build backend becomes the stable default.
```

Comment:
Should we also test `uv init --package --preview`?

I'd also expect, though it could be a future change, for `uv init --preview` to package by default.

Reply:
Changing `--package` is a later stage, for now I'm focussing on the build backend itself.